test-support/assertions/close-to.js

```diff
@@ -0,0 +1,15 @@
+export default function(context, actual, target, delta, message) {
```

Comment:
Can you rename target to expected? Just to make it obvious. `target` works for cases where the same param accepts multiple type values.

Reply:
done